app/views/external_users/admin/external_users/index.html.haml

```diff
@@ -4,8 +4,9 @@
 = render partial: 'layouts/header', locals: { page_heading: t('.page_heading', provider_name: "#{current_user.provider.name}") }
 
 = content_for :search_html_block do
-  .form-group
-    = govuk_link_to t('.new_user'), new_external_users_admin_external_user_path
+  .div{ class: "govuk-!-margin-bottom-6" }
+    .form-group
```

Comment:
Does it work if the class is added to the `form-group` instead of wrapping in a new `div`?